src/components/DocumentUtility/DocumentUtility.tsx

```diff
@@ -7,7 +7,8 @@ import { SvgIcon, SvgIconQRCode } from "../UI/SvgIcon";
 import { Printer, Mail, Download } from "react-feather";
 import { ButtonIconWhiteBlue } from "../UI/Button";
 import { Popover, OverlayTrigger } from "react-bootstrap";
-import { QRCode } from "react-qr-svg";
+import QRCode, { ImageSettings } from "qrcode.react";
```

Comment:
want to remove `react-qr-svg`? if its no longer used anywhere in the project

Reply:
Do we need to modify the credits then😮? [this one](https://github.com/TradeTrust/tradetrust-website/blob/master/CREDITS.md)

Reply:
mmm u can do so, maybe in a separate pr